lite/examples/digit_classifier/ios/DigitClassifier/DigitClassifier.swift

```diff
@@ -99,10 +99,7 @@ class DigitClassifier {
             print("Failed to convert the image buffer to RGB data.")
             return
         }
-
-        // Allocate memory for the model's input `Tensor`s.
-        try self.interpreter.allocateTensors()
-
+        
```

Comment:
Looks like there's a whitespace addition here, can you remove it?

Reply:
Thanks your review.
I removed it.